db/migrate/20210730190522_remove_required_from_questions.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveRequiredFromQuestions < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :questions, :required, :boolean
```

Comment:
This became obsolete when we introduced per-loan requiredness

Reply:
thanks!